rb/ketama.py

```diff
@@ -3,9 +3,18 @@
 
 from bisect import bisect
 
+from rb.utils import text_type, integer_types, bytes_type
+
 
 def md5_bytes(key):
-    return map(ord, hashlib.md5(key).digest())
+    if isinstance(key, text_type):
+        k = key.encode("utf-8")
```

Comment:
Decided to try and be a little extra safe here on the key

Reply:
If we need to do this again we should probably make a `key_to_bytes` helper.

Reply:
Yeah agree